### ISSUE TITLE
plan expansion of accessibility scanning

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#1869]
per NOAA May '26 request.  

Specifically, we will be increasing the number of axe-core rules that we scan for, along the lines of [this request](https://docs.google.com/document/d/1hRvy4iJDvKJfmJlOXI5itKTGotYIRJW5fvifHZsS1XY/edit?tab=t.0#heading=h.vejko09o9c4c).  [Here's](https://github.com/GSA/site-scanning-documentation/blob/main/data/Axe-core-rules.csv) the current axe-core rules we scan for.  

I think we would want to include the results in the full accessibility details [file](https://api.gsa.gov/technology/site-scanning/data/weekly-snapshot-accessibility-details.json) for sure, but we still need to validate whether we will add these to the counts in the accessibility_violations field, create a second field to count them in, or just not have them reflected in any of the fields in the site scanning snapshots.

**Comment by gbinal** [comment]
This is done.  Good work again, @ethangardner
